UPDATE CONTRIBUTEING.MD

j-tesla opened the pull request:
Added archive links to the articles

thealphadollar commented:
@j-tesla Hey, thanks for the pull request.

I would like to know why do you think this is a required change?

j-tesla commented:
> @j-tesla Hey, thanks for the pull request.
> 
> I would like to know why do you think this is a required change?

@thealphadollar One of the links *( [https://code.likeagirl.io/useful-tips-for-writing-better-git-commit-messages-808770609503](https://code.likeagirl.io/useful-tips-for-writing-better-git-commit-messages-808770609503) )* is not working anymore as the medium story is closed by it's author. To make the article accessible, I added the archive link. To cover any similar issue that may occur later, I added the archives to other articles too.

thealphadollar commented:
Yeah, thanks for the explanation. Highly appreciated, and also thanks for the PR.
